src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -50,6 +50,7 @@
 #include <cassert>
 #include <limits>
 #include <memory>
+#include <optional>
```

Comment:
Actually, I fixed this previously for different reasons, to simplify the evaluation of which ledgers to keep online. Since that's driven partly by SOCI's return value of a query, which is boost::optional, I ditched std::optional. Anyway, all's well that ends well.

Reply:
Ah, thanks for pointing that our.  You were dealing with a SOCI return value.  That makes all the sense in the world.  Sorry for the noise.